docs/user/set-up/_index.md

```diff
@@ -39,59 +40,27 @@ For sample configuration files, refer to the Grafana Cloud Kubernetes quick star
 
-### Windows
+### Docker
 
-Use the [Windows Installer]({{< relref "./install-agent-on-windows.md" >}})
+Refer to [Install Grafana Agent on Docker]({{< relref "./install-agent-docker.md" >}})
 
-### Docker container
 
-```
-docker run \
-  -v /tmp/agent:/etc/agent/data \
-  -v /path/to/config.yaml:/etc/agent/agent.yaml \
-  grafana/agent:v0.24.2
-```
+### Windows
 
-Replace `/tmp/agent` with the folder you wish to store WAL data in. WAL data is
-where metrics are stored before they are sent to Prometheus. Old WAL data is
-cleaned up every hour, and will be used for recovering if the process happens to
-crash.
+Refer to [Install Grafana Agent on Windows]({{< relref "./install-agent-on-windows.md" >}})
 
-To override the default flags passed to the container, add the following flags
-to the end of the `docker run` command:
+### Binary
 
-- `--config.file=path/to/agent.yaml`, replacing the argument with the full path
-  to your Agent's YAML configuration file.
+Refer to [Install the Grafana Agent binary]({{< relref "./install-agent-binary.md" >}})
 
-- `--metrics.wal-directory=/tmp/agent/data`, replacing `/tmp/agent/data` with
-  the directory you wish to use for storing data. Note that `/tmp` may get
-  deleted by most operating systems after a reboot.
+### macOS
 
-Note that using paths on your host machine must be exposed to the Docker
-container through a bind mount for the flags to work properly.
+Refer to [Install Grafana Agent on macOS]({{< relref "./install-agent-macos.md" >}})
 
-### Install locally
+### Grafana Cloud
 
-Our [Releases](https://github.com/grafana/agent/releases) page contains
-instructions for downloading static binaries that are published with every release.
-These releases contain the plain binary alongside system packages for Windows,
-Red Hat, and Debian.
+Use the Grafana Agent [Kubernetes quickstarts](https://grafana.com/docs/grafana-cloud/kubernetes/agent-k8s/) or follow instructions for installing the Grafana Agent in the [Walkthrough](https://grafana.com/docs/grafana-cloud/quickstart/agent_linuxnode/).
 
 ### Tanka
 
-We provide [Tanka](https://tanka.dev) configurations in our [`production/`](https://github.com/grafana/agent/tree/main/production/tanka/grafana-agent) directory.
-
-### Community Projects
-
-Below is a list of community lead projects for working with Grafana Agent. These projects are not maintained or supported by Grafana Labs.
-
-#### Helm (Kubernetes Deployment)
-
-A publically available release of a Grafana Agent Helm chart is maintained [here](https://github.com/DandyDeveloper/charts/tree/master/charts/grafana-agent). Contributions and improvements are welcomed. Full details on rolling out and supported options can be found in the [readme](https://github.com/DandyDeveloper/charts/blob/master/charts/grafana-agent/README.md).
-
-This *does not* require the Grafana Agent Operator to rollout / deploy.
-
-#### Juju (Charmed Operator)
```

Comment:
Are we removing both of these? I'm not certain about how often these projects are updated, but it might be a good sign to recognize community efforts.

Reply:
@rfratto can you weigh in? if we keep--maybe they don't belong in a set up section but maybe the landing page overview..?

Reply:
Can we move the community projects section to the top-level README.md of the project? (e.g., outside of the hosted docs but still in the repo itself)